doc/src/tutorial/calculus.rst

```diff
@@ -81,6 +81,19 @@ know how to compute the derivative of an expression (for example, if it
 contains an undefined function, which are described in the :ref:`Solving
 Differential Equations <tutorial-dsolve>` section).
 
+Derivatives of unspecified order can be created using tuple ``(x, n)`` where
+``n`` is the order of the derivative with respect to ``x``.
+
+    >>> m, n = symbols('m n', integer=True)
+    >>> a, b = symbols('a b', real=True, constant=True)
```

Comment:
I don't think SymPy supports constant=True

Reply:
Yup changed that, Thanks1